roles/satellite-clone/tasks/restore.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Restore mongo data
   command: "tar --selinux --overwrite -xf {{ backup_dir }}/mongo_data.tar.gz -C /"
-  when: standard_backup
+  when: rhel_migration or standard_backup
```

Comment:
I am not sure why we should mirror katello-restore.  katello-restore doesn't handle migration or support finite defined use cases with proper error handling as satellite-clone. As long as satellite-clone makes cloning/migration easier, it is good.  I like to keep this tool as simple as possible to use and maintain.  In future, if there is a need to merge satellite-clone with katello-restore we can definitely think about it, but not right now.

Having said that:
Now pulp backup is the only file which is not dependent on the scenarios and we are rightfully handling it with:
```sh
- name: Restore pulp data
  command: "tar --selinux --overwrite -xf {{ backup_dir }}/pulp_data.tar --exclude=var/lib/pulp/katello-export -C /"
  when: pulp_data.stat.exists
```
All other tasks are clearly scenario dependent (see below).    It doesn't make sense to change mongo check until we change all scenarios listed below, which is not the intention of this PR.  If you feel strongly that this is the way to go, please create a new issue and we can brainstorm on how best to handle this.
```sh
- name: Restore postgresql data
  command: tar --selinux --overwrite -xf {{ backup_dir }}/pgsql_data.tar.gz -C /
  when: standard_backup

- include: restore_psql_dump.yml
  when: rhel_migration or online_backup

- name: Restore mongo data
  command: "tar --selinux --overwrite -xf {{ backup_dir }}/mongo_data.tar.gz -C /"
  when: rhel_migration or standard_backup

- include: restore_mongo_dump.yml
  when: online_backup
```